### PR TITLE
CEDS-2027 Add GDS Design system to Locations page

### PR DESCRIPTION
**Pull request by PauloPortugal**

### CODE REVIEW

File: app/views/location.scala.html [hunk 1]
```diff
@@ -17,27 +17,38 @@
...
+    title = Title("location.question"),
+    backButtonTitle = messages("site.back"),
+    backButtonUrl = controllers.routes.MovementDetailsController.displayPage.url) {
+    @formHelper(action = controllers.routes.LocationController.saveLocation(), 'autoComplete -> "off") {
 
-        @components.back_link(routes.MovementDetailsController.displayPage())
+        @errorSummary(form.errors)
 
-        @components.error_summary(form.errors)
+        @sectionHeader(messages("movement.sectionHeading", request.answers.`type`.toString.toLowerCase.capitalize, consignmentReference.getOrElse("")))
```
Comment (SteveSugden) [inline app/views/location.scala.html]: Patryk doesn't like capitalisation in message keys (I know you haven't changed this but....)

**Comment by PauloPortugal** [comment]
please test again

**Comment by PauloPortugal** [comment]
Please test this

**Comment by PauloPortugal** [comment]
Please test this again

**Comment by PauloPortugal** [comment]
test this again please

**Comment by PauloPortugal** [comment]
test this please

**Comment by PauloPortugal** [comment]
@SteveSugden 

> LGTM. Maybe re-introduce the assertion on the back button text as it seems to matter these days (i.e. different on different pages)

Done

**Pull request merged**